src/kili/mutations/asset/__init__.py

```diff
@@ -40,6 +40,7 @@ def append_many_to_dataset(
         project_id: str,
         content_array: Optional[List[str]] = None,
         external_id_array: Optional[List[str]] = None,
+        id_array: Optional[List[str]] = None,
```

Comment:
IdArray is not the ids of the assets so this seems very confusing. Can't you give another name ?

Reply:
and if I understand well, it is for internal use only. @PierreLeveau should be add an underscore before its name ?

Reply:
like `(for internal use only)`